### PR TITLE
Replace bazel_deps with rules_jvm_external

### PR DESCRIPTION
**Pull request by borkaehw**
Replace bazel_deps with rules_jvm_external.

### CODE REVIEW

File: setup-tools.sh [hunk 1]
```diff
@@ -1,17 +1,4 @@
 #!/bin/bash -e
```
Comment (borkaehw) [inline setup-tools.sh]: Not sure if we still need this file.

**Review by borkaehw — COMMENTED** [review]
I like how neat it becomes. Buildozer works fine as well.

File: setup-tools.sh (hunk 1 above)
Comment (SrodriguezO) [reply setup-tools.sh]: I think we should remove it; it's just making an empty directory now

**Review by SrodriguezO — APPROVED** [review]
LGTM :) 

Test using this commit before merging please :) 

(And remove `setup-tools.sh`)

**Review by JaredNeil — APPROVED** [review]

**Pull request merged**